cmd/stack.go

```diff
@@ -111,20 +110,28 @@ func newStackCmd() *cobra.Command {
 			if rescnt == 0 {
 				fmt.Printf("    No resources currently in this stack\n")
 			} else {
-				fmt.Printf("    %-48s %s\n", "TYPE", "NAME")
+				prefix := "    "
+				table := []cmdutil.TableRow{}
+				table = append(table, cmdutil.TableRow{Columns: []string{prefix + "TYPE", "NAME"}})
```

Comment:
Might be slightly nicer to have an `prefix` capability for the table. Since we only use this in this one place, not a big deal.

Reply:
Yeah.  I didn't want to get into design-heck :)   High order bit was the dynamic columns.  :)

Reply:
Sorry, I'm getting all carried away with the coolness of ASCII art CLI tables now 😉

Reply:
Have broken out the prefix.